File: 1-Iniciante/1066-Pares,Ímpares,Positivos_e_Negativos-1/1066.cs
using System;

public class HelloWorld
{
    public static void Main(string[] args)
    {

        
        double par=0;
        double pos=0;
        double neg=0;
        
        for(int i=0;i<5;i++){
            double x =double.Parse(Console.ReadLine());
                if (x%2==0){
                    par++;
                }
                if (x<0){
                    neg++;
                }
                if (x>0){
                    pos++;
                }
            }
        Console.WriteLine($"{par} valor(es) par(es)");
        Console.WriteLine($"{5-par} valor(es) impar(es)");
        Console.WriteLine($"{pos} valor(es) positivo(s)");
        Console.WriteLine($"{neg} valor(es) negativo(s)");
        
    }
}

File: 1-Iniciante/1066-Pares,Ímpares,Positivos_e_Negativos-1/1066.py
x=[int(input()),int(input()),int(input()),int(input()),int(input())]
par=0
pos=0
neg=0
for i in x:
    if i%2==0:
        par+=1
    if i<0:
        neg+=1
    elif i>0:
        pos+=1
print(f"{par} valor(es) par(es)")
print(f"{len(x)-par} valor(es) impar(es)")
print(f"{pos} valor(es) positivo(s)")
print(f"{neg} valor(es) negativo(s)")